IronFoundry.Container.Test/Utilities/FileSystemEffectiveAccessComputerTests.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.IO;
+using System.Security.AccessControl;
+using System.Security.Principal;
+using Xunit;
+
+namespace IronFoundry.Container.Utilities
+{
+    public class FileSystemEffectiveAccessComputerTests : IDisposable
+    {
+        TempFile TestFile { get; set; }
+        FileSystemEffectiveAccessComputer EffectiveAccess { get; set; }
+        IdentityReference CurrentIdentity { get; set; }
+
+        public FileSystemEffectiveAccessComputerTests()
+        {
+            TestFile = new TempFile(Path.GetTempPath());
+            EffectiveAccess = new FileSystemEffectiveAccessComputer();
+            CurrentIdentity = WindowsIdentity.GetCurrent().User;
+        }
+
+        public void Dispose()
+        {
+            TestFile.Dispose();
+        }
+
+        public class ComputeAccess : FileSystemEffectiveAccessComputerTests
+        {
+            [Fact(Skip = "Fails on domain joined machines")]
```

Comment:
That sounds promising.  Let's just plan to spend a little time and investigate on my domain joined machine on Monday. For now, let's just skip these tests.